git-link.el

```diff
@@ -494,6 +496,25 @@ return (FILENAME . REVISION) otherwise nil."
 	  dirname
 	  commit))
 
+(defun git-link-sourcegraph (hostname dirname filename branch commit start end)
+    (let ((line-or-range (if end (format "%s-%s" start end) start))
+          (branch-or-commit (or branch commit))
+          (dir-file-name (directory-file-name dirname)))
+      (format "https://%s/%s@%s/-/blob/%s#L%s"
+              hostname
+              dir-file-name
+              branch-or-commit
+              filename
+              line-or-range)))
```

Comment:
`line-or-range` may be `nil`. For example, if you're linking from a magit-log or magit-status buffer. 

Currently it'll try to link to `#Lnil`

Reply:
Good point this should be fixed now.